integration-test/concat-dependencies/build-and-push.sh

```diff
@@ -19,15 +19,19 @@ fi
 
 full_tag="mediachain/concat-dependencies:${image_tag}"
 
-docker build -t ${full_tag} .
+if [ "${concat_ref}" == "" ]; then
+    concat_ref="master"
+fi
+
+docker build --build-arg concat_ref=${concat_ref} -t ${full_tag} .
 status=$?
 if [ ${status} -ne 0 ]; then
     echo "Error building image, not uploading to docker hub"
     exit ${status}
 fi
 
 echo "image \"${full_tag}\" built successfully, pushing to docker hub"
-docker push ${full_tag}
+#docker push ${full_tag}
```

Comment:
uhm the script is called build and push and the push is commented out.
Maybe we should have a separate script for build and push?